src/patternfly/components/InputGroup/input-group.scss

```
@@ -27,6 +27,10 @@
   width: 100%;
   background-color: var(--pf-c-input-group--BackgroundColor);
 
+  &.pf-m-plain {
+    --pf-c-input-group--BackgroundColor: transparent; // update at breaking change
```

wdyt about this? just so we aren't left wondering what we agreed to update it to?  😅

```suggestion
    --pf-c-input-group--BackgroundColor: transparent; // make default at breaking change
```